partner_statement/report/current_statement.py

```diff
@@ -0,0 +1,160 @@
+# Copyright 2018 ForgeFlow, S.L. (https://www.forgeflow.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, models
+
+
+class CurrentStatement(models.AbstractModel):
+    """Model of Current Statement"""
+
+    _inherit = "statement.common"
+    _name = "report.partner_statement.current_statement"
+    _description = "Partner Current Statement"
+
+    def _display_lines_sql_q1(self, partners, date_end, account_type):
```

Comment:
```suggestion
    def _display_current_lines_sql_q1(self, partners, date_end, account_type):
```
same for the other methods